src/platform/x11.rs

```diff
@@ -138,9 +138,16 @@ impl EventLoopBuilderExtX11 for EventLoopBuilder {
 /// Additional methods on [`Window`] that are specific to X11.
 ///
 /// [`Window`]: crate::window::Window
-pub trait WindowExtX11 {}
+pub trait WindowExtX11 {
+    fn set_override_redirect(&self, value: bool);
```

Comment:
Please add documentation to this method.

Reply:
done